ext/nnstreamer/meson.build

```diff
@@ -1,2 +1,2 @@
-subdir('tensor_decoder')
-subdir('tensor_filter')
+subdir('tensor_decoder')
```

Comment:
I can't find and understand the changes in this file.

Reply:
It has gone thru dos2unix. It shuold've been in another commit.